src/core/core.zig

```diff
@@ -0,0 +1,8 @@
+pub usingnamespace @import("pubkey.zig");
```

Comment:
the single lib with all namespaces is the simplest approach rn - moving these across files just adds another layer of indirection

while the prs you mentioned do bring up removing it - i think making changes once it is removed (and not before) will be the best approach